### PR TITLE
 ISY-183 Migrationsleitfaden ergaenzen Bridgemodule

### PR DESCRIPTION
**Pull request by dkurowski-materna**

### CODE REVIEW

**Review by kollmana — DISMISSED** [review]
Can not review picture IFS3_calls.dn.png due to problems with LFS

**Review by kollmana — DISMISSED** [review]
~~Can not review picture IFS3_calls.dn.png due to problems with LFS~~

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # 3.0.0
+- [isyfact-standards-doc] Migrationsleitfaden zur Entfernung der Bridge-Module
```
Comment (huebnercl) [inline CHANGELOG.md]:
```suggestion
- `ISY-183`: [isyfact-standards-doc] Migrationsleitfaden zur Entfernung der Bridge-Module
```

File: isyfact-standards-doc/src/docs/antora/modules/changelog/pages/migrationsleitfaden.adoc [hunk 2]
```diff
@@ -11,6 +11,46 @@ Die auf dieser Seite aufgeführten Hinweise sollen IsyFact Anwendungsentwicklern
...
+Die drei in IsyFact 2 eingeführten Bridge-Module
+
+* `isy-serviceapi-core-bridge`
+* `isy-serviceapi-sst-bridge`
+* `isy-exception-sst-bridge`
+
+entfallen in IsyFact 3.
+In einer Anwendung, die bisher Bridge-Module verwendet und auf IsyFact 3 umgestellt werden soll, müssen daher Anpassungen vorgenommen werden.
+Die Tabelle <<table-bridge-module>> zeigt für jedes der drei Bridge-Module, durch welches Modul es ersetzt wird.
+Dabei ist zu beachten, dass zwei der Ersatzmodule in IsyFact 1-Versionen verwendet werden.
```
Comment (huebnercl) [inline isyfact-standards-doc/src/docs/antora/modules/changelog/pages/migrationsleitfaden.adoc]:
```suggestion
Dabei ist zu beachten, dass zwei der Ersatzmodule eine Abhängigkeit zu IsyFact-1.x haben.
```

**Review by huebnercl — CHANGES_REQUESTED** [review]
however found some small changes/suggestions...

**Comment by as-materna** [comment]
@huebnercl accepted suggestions and resolved conflicts. Please approve and merge.

**Review by huebnercl — APPROVED** [review]

**Pull request merged**